libs/watt/src/lib/components/textarea-field/watt-textarea-field.component.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgClass, NgIf } from '@angular/common';
+import {
+  Component,
+  Input,
+  forwardRef,
+  ViewEncapsulation,
+  HostBinding,
+  inject,
+  ElementRef,
+} from '@angular/core';
+import { ControlValueAccessor, FormsModule, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { WattFieldComponent } from '../field/watt-field.component';
+
+@Component({
+  standalone: true,
+  imports: [NgClass, FormsModule, WattFieldComponent, NgIf],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => WattTextAreaFieldComponent),
+      multi: true,
+    },
+  ],
+  selector: 'watt-textarea-field',
+  styleUrls: ['./watt-textarea-field.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  template: `<watt-field [label]="label">
+    <textarea
+      [attr.placeholder]="placeholder"
+      [value]="value"
+      [(ngModel)]="model"
+      [disabled]="isDisabled"
+      (ngModelChange)="onChange($event)"
+      [required]="required"
+    ></textarea>
+    <ng-content />
```

Comment:
question: is there a use case where this can be necessary? In the text field component this can be used for a suffix button, but maybe that's not relevant here?

Reply:
You are right, i forgot to remove it ! :-)